api/swagger.yaml

```diff
@@ -9367,7 +9374,7 @@ paths:
                   InUse: false
                   Shared: true
                   Size: 0
-                  CreatedAt: "2021-06-28T13:31:01.474619385Z"
+                  CreatedAt: "2021-06-28T13:31:01.464619385Z"
```

Comment:
Why did this line change?

Reply:
That's a typo, I'l be more careful about proofreading.